interpreter/cling/include/cling/Interpreter/InterpreterCallbacks.h

```diff
@@ -125,6 +125,8 @@ namespace cling {
     virtual bool LookupObject(const clang::DeclContext*, clang::DeclarationName);
     virtual bool LookupObject(clang::TagDecl*);
 
+   virtual void LazyFunctionCallback(std::string mangled_name) {}
```

Comment:
We should use more meaningful name. What about SymbolUndefined(std::string mangled_name, std::string dynamicLibrary)